src/patternfly/components/DataList/data-list.scss

```diff
@@ -204,7 +218,36 @@
   }
 
   &.pf-m-drag-over {
+    position: relative;
```

Comment:
Here's an example of divs: https://patternfly-react-pr-6328.surge.sh/components/drag-and-drop

Reply:
> I'd like this class to be able to be applied to more than just data lists (for example divs or other ul/lis). I don't think I should have to make those datalists in order to get the styling. Also, making these changes may break the existing drag and drop implementation. Could you instead make a DragDrop/drag-drop.scss with some pf-m-droppable and pf-m-draggable that will work on any ul/div and li/div including DataList?

@redallen if the thing you're adding `.pf-m-draggable` to is unstyled, then yeah that would work. The problem comes when you add `.pf-m-draggable/droppable` to something that already has a border or box shadow or whatever styles we're adding to `.pf-m-draggable/droppable`, and those styles will conflict. I'm not sure how we would get around needing to style individual components for the draggable and droppable elements in existing components to avoid those style conflicts.

Reply:
> Couldn't you write [pf-c-*].pf-m-draggable

That may work, I think we'll still end up with conflicting styles.

> use `!important`

Same, there could easily be other conflicting styles we'll run into. For example, currently I'm using pseudo elements on the draggable and droppable elements to draw the borders and overlay for the droppable element. If those pseudo elements are already in use for component styles, when we add the draggable/droppable classes, any existing pseudo element styles will disappear. I think we'll just end up playing whack-a-mole with conflicts as users make different things draggable/droppable.

Reply:
Hm, I think we need not play whack-a-mole if we give users guidelines on what styles of theirs may be overriden and try to be as permissible as possible given the design. I think the choice of keeping your markup but dealing with overides or keeping your styles but dealing with markup is a fair one given we document it correctly.